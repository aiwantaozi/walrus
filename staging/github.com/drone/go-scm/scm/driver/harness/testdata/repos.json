[
  {
    "id": 9,
    "parent_id": 16,
    "uid": "demo",
    "path": "px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/demo",
    "description": "",
    "is_public": false,
    "created_by": 10,
    "created": 1675210116158,
    "updated": 1675389801066,
    "default_branch": "main",
    "fork_id": 0,
    "num_forks": 0,
    "num_pulls": 4,
    "num_closed_pulls": 0,
    "num_open_pulls": 1,
    "num_merged_pulls": 3,
    "git_url": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/demo.git"
  },
  {
    "id": 11,
    "parent_id": 16,
    "uid": "thomas",
    "path": "px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/thomas",
    "description": "Playground project for Thomas",
    "is_public": true,
    "created_by": 10,
    "created": 1675458112548,
    "updated": 1675458112548,
    "default_branch": "main",
    "fork_id": 0,
    "num_forks": 0,
    "num_pulls": 0,
    "num_closed_pulls": 0,
    "num_open_pulls": 0,
    "num_merged_pulls": 0,
    "git_url": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/thomas.git"
  }
]