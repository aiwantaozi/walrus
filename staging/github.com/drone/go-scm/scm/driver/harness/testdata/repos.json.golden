[
    {
        "ID": "9",
        "Namespace": "px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/demo",
        "Name": "demo",
        "Branch": "main",
        "Private": true,
        "Clone": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/demo.git",
        "CloneSSH": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/demo.git",
        "Link": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/demo.git"
    },
    {
        "ID": "11",
        "Namespace": "px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/thomas",
        "Name": "thomas",
        "Branch": "main",
        "Private": false,
        "Clone": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/thomas.git",
        "CloneSSH": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/thomas.git",
        "Link": "https://qa.harness.io/code/git/px7xd_BFRCi-pfWPYXVjvw/default/codeciintegration/thomas.git"
    }
]